client/src/components/statistics/SeriesChart.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { BarChart, Bar, XAxis, YAxis, CartesianGrid, Tooltip, Legend, Label } from 'recharts';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faArrowLeft, faArrowRight } from '@fortawesome/free-solid-svg-icons';
+
+class SeriesChart extends React.Component {
+  state = {
+    page: 0,
+    pageSize: 5,
+  }
+
+  handlePrevPage = () => {
+    this.setState((prevState) => ({ page: prevState.page - 1 }));
+  }
+
+  handleNextPage = () => {
+    this.setState((prevState) => ({ page: prevState.page + 1 }));
+  }
+
+  render() {
+    const { series, dataKey } = this.props;
+    const { page, pageSize } = this.state;
+    const data = series.slice(page * pageSize, (page + 1) * pageSize).map((s) => ({ name: s.name, [dataKey]: s[dataKey] }));
```

Comment:
It pagination worked easily because you used `/series` with 10 static entities and 1 request only.
What will happen with real data is that you'll do `/series?pageNumber=1`, you'll get 10 entities.
In the next page, you'll have to make another API request for 10 different entities.
I think the pagination should be handled outside this chart component (probably in statistics component). The chart will handle the chart logic only and you'll inject the updated `series` array to it every time.

Consider using `Pagination` component, it'll handle most of `handleNextPage`, `handlePrevPage`, `disabled` logic, will show total amount of pages..